sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -994,10 +994,10 @@ public virtual Response<ExtractKeyPhrasesResult> ExtractKeyPhrases(string inputT
                     case 200:
```

Comment:
I don't think we should be raising exceptions for case `200 OK`. Especially not when we have different behavior patterns. i.e The same scenario for a single input would raise an exception but wouldn't for a batch operation.

For example, If a user was to send ` "code": "InvalidCountryHint"`  for a single input method would raise an exception whereas for a batch method input would only result in a `TextAnalyticsError`.  Is would be difficult for the users to navigate as to when can it expect an exception vs an error.

Reply:
I see your point, but I think this is the right thing to do from a .NET perspective.  It is idiomatic to .NET to throw an exception on an error case.  In the case of a single input, if there is an error, there is nothing that has succeeded, and we should therefore throw an error.  It's different in the case of multiple inputs because there are errors mixed in with success results, and the success results are useful -- callers can write a Linq expression to remove the errors and then proceed with their analysis.  It may be different for Java, but it would be nice to have consistency across the two languages.